site/components/Donation/index.js

```diff
@@ -18,6 +27,14 @@ const Donation = props => {
         ExtraLifeService.getTeam()
             .then(response => setTeam(response.data))
             .catch(err => console.log(err));
+
```

Comment:
This will cause 3 renders. Instead, you can create a ExtraLife object that holds the team, donations, and donors. 

Promise.all([... your 3 calls]).then((resp1, resp2, resp2) => setExtraLifeObject({team: resp1, donation: resp3, donors: resp2});

Nothing wrong here, but could decrease number of renders :)